projects/SelfTest/UsageTests/Misc.tests.cpp

```diff
@@ -491,3 +492,34 @@ TEMPLATE_TEST_CASE_SIG("#1954 - 7 arg template test case sig compiles", "[regres
 }
 
 }} // namespace MiscTests
+
+#if defined(CATCH_PLATFORM_WINDOWS)
```

Comment:
The test in v3 only checks with MSVC. Why the change?

Reply:
I certainly didn't mean to change it.